gold-phone-input.html

```diff
@@ -88,7 +88,7 @@
       <label hidden$="[[!label]]">[[label]]</label>
 
       <div class="horizontal layout">
-        <span>+<span>[[countryCode]]</span></span>
+        <span hidden="[[noCountryCode]]">+<span>[[countryCode]]</span></span>
```

Comment:
Yeah, that's a much better way to implement it!

I removed the additional span as well since v1.2 supports concatenating :)